### ISSUE TITLE
Scale of objects is 1000x bigger.

### ISSUE THREAD

**Issue by ipepe** [slugdev/stltostp#1]
@slugdev Can You confirm that all dimensions are 1000x bigger after conversion? We did notice that when opening converted STL files in SolidEdge.

**Comment by RTnhN** [comment]
Yep, it was the same for me too @ipepe . When I imported the model into my CAD software, I just needed to scale the part down.

**Comment by usmanredsoft-zz** [comment]
“Please fix error with multiplying by 1000 (Scale of objects is 1000x bigger.)"
